docs/source/reference/yaml-spec.rst

```diff
@@ -331,3 +331,28 @@ Available fields:
 
       # Demoing env var usage.
       echo Env var MODEL_SIZE has value: ${MODEL_SIZE}
+
+Experimental
+------------
+
+In additional to the above fields, SkyPilot also supports the following experimental fields in the task YAML:
```

Comment:
Should we add a note box saying something along the lines of: "Experimental features and API may be changed or removed without any notice."

Reply:
Good idea! Added.